java/connector-node/risingwave-connector-service/src/main/java/com/risingwave/connector/source/core/DbzSourceHandler.java

```diff
@@ -29,9 +34,12 @@
 public class DbzSourceHandler implements SourceHandler {
     static final Logger LOG = LoggerFactory.getLogger(DbzSourceHandler.class);
 
+    private Map<Long, String> replicationSlotMap;
```

Comment:
I think we can use a singleton class to store the slot name for each source, then you don't need to pass the replicationSlotMap all the way of the call stack. Just access the singleton class is enough, which can make the code cleaner.